### PR TITLE
fix: tagging au-12 controls

### LINKED ISSUE #764: fix: Tagging AU-12 security controls to the resources
This adds AU-12 security controls to resources that has logging enabled.

### PR DESCRIPTION
**Pull request by anoopsidhu-ssc**
closes #764

### CODE REVIEW

File: examples/landing-zone-v2/configconnector/tier3/client-project-iam/pubsec-declarative-toolkit.code-workspace
Comment (cartyc) [inline examples/landing-zone-v2/configconnector/tier3/client-project-iam/pubsec-declarative-toolkit.code-workspace:1]: Does this file need to be here or can it be removed?

**Review by cartyc — CHANGES_REQUESTED** [review]

**Review by fmichaelobrien — APPROVED** [review]
/lgtm after comments resolved

**Review by fmichaelobrien — DISMISSED** [review]
/lgtm

File: solutions/client-landing-zone/client-folder/firewall-policy/rules/iap.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #########
 # Delegate to host project, ingress IAP traffic (firewall) from IP range 35.235.240.0/20 to VPC resources in shared VPC network
 # AC-3(9), AC-4, AC-4(21), SC-7(5), SC-7(8), SC-7(9), SC-7(11) - All connections to or from virtual machine instances are allowed/denied via firewall rules configured in shared VPC network within host project or firewall policies in parent folders based on least-privilege principle. Each firewall rule applies to incoming(ingress) or outgoing(egress) connections, not both.
+# AU-12 - Disable Logging for firewall policies
```
Comment (lucstjean-ssc) [inline solutions/client-landing-zone/client-folder/firewall-policy/rules/iap.yaml:17]: I wondered why we disabled logging here until I read `# logging not supported for goto_next rules` on line 30. Any chance we can duplicate or move this comment after line 17? If so, there's also a bunch of other files that this comment would apply to.

Comment (anoopsidhu-ssc) [reply solutions/client-landing-zone/client-folder/firewall-policy/rules/iap.yaml]: great suggestion thanks alain and luc. Removed au-12 tagging for go to next rules

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

**Review by alaincormier-ssc — DISMISSED** [review]

**Review by fmichaelobrien — DISMISSED** [review]
/lgtm

**Review by davelanglois-ssc — CHANGES_REQUESTED** [review]
I would like to see the securitycontrols.md file updated in the affected packages

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Review by davelanglois-ssc — APPROVED** [review]

**Review by cartyc — APPROVED** [review]

**Pull request merged**